Gems/AWSCore/Code/CMakeLists.txt

```diff
@@ -189,4 +189,22 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
     endif()
 endif()
 
+ly_add_target_files(
+    TARGETS
+        AWSCore
+    FILES
+        ${CMAKE_CURRENT_SOURCE_DIR}/Tools/ResourceMappingTool/resource_mapping_schema.json
```

Comment:
files are already relative to `CMAKE_CURRENT_SOURCE_DIR` so you can just pass `Tools/ResourceMappingTool/resource_mapping_schema.json` here (same for bellow

Reply:
Tried, but it throws following error:
```
Copying AWSCore.Tests runtime dependencies to output...
CMake Error at E:/Lumberyard/o3de/windows_vs2019_AT/runtime_dependencies/profile/AWSCore.Tests.cmake:17 (file):
  file SIZE requested of path that is not readable:

    Tools/ResourceMappingTool/resource_mapping_schema.json
```

Reply:
ah, my bad, fine to leave it as you had it, I will create a ticket to allow to make that path relative and cleanup